mqtt2influxdb/mqtt2influxdb.py

```diff
@@ -106,6 +111,12 @@ def _on_mqtt_message(self, client, userdata, message):
                           'time': datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'),
                           'tags': {},
                           'fields': {}}
+                if 'base64decode' in self._config.keys():
```

Comment:
Using .keys () is not necessary, just 
if 'base64decode' in self._config:

Small story:
My colleague, who used the same code in his script and ran the script for over 5 hours, I removed ".keys ()" from it and ran only lightly for 1 hour.